lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -470,14 +470,16 @@ def create_patient(record: REDCapRecord) -> tuple:
         'preferred': True # Assumes that the project language is the patient's preferred language
     }]
 
-    # Use the UW NetID to generate patient id if it's available.
+    # Use the UW NetID and domain to generate patient id, if it's available.
     # This is the stable identifier that can allow us to match UW reopening
     # participants.
     #   -Jover, 04 September 2020
     net_id = record.get('netid')
-    net_id = net_id.strip().lower() if net_id else None
+    net_id = net_id.split('@')[0].strip().lower() if net_id else None
+    username = f'{net_id}@washington.edu' if net_id else None
```

Comment:
Hmm, after mentioning doctests, I just realized that this code will turn `xxx@gmail.com` → `xxx@washington.edu`. I think we want to avoid that. I know we have (had?) enrollments with @gmail.com addresses for the NetID. Maybe doctests with edge cases for this behaviour would be best after all!

Reply:
This is true. The study team seems diligent in cleaning up the gmails etc (I don't see any in their reports now) but it'd be reasonable to detect a non-uw email and hash that as entered. I think the uw email domains are @u.washington.edu and @uw.edu

Reply:
Yep, I believe those are the two domains accepted. Instead of using the non-UW address, perhaps just fall back to hashing by the standard PII tuple?